README.md

```diff
@@ -24,32 +25,31 @@ There is an `internal` subdirectory that contains service protocol buffers,
 plus packages required for connectivity to make API calls. App Engine apps
 should not directly import any package under `internal`.
 
-## Updating a Go App Engine app
+## Upgrading an App Engine app to the flexible environment
```

Comment:
I was going to say that _some_ parts work, but it's really just `appengine.Main`.

Will do.